procedure/ferie-permessi-malattia.md

```diff
@@ -12,22 +15,43 @@ competenza dell'anno precedente ma tuttavia ancora godibili.
 
 ## Ferie
 
-Sono previsti 22 giorni di ferie l'anno. Tutti i membri del team devono comunicare, tramite chat o
+Sono previsti 22 giorni di ferie l'anno (176 ore). Tutti i membri del team devono comunicare, tramite chat o
 anche a voce, possibilmente almeno due settimane prima, i giorni di ferie di cui si desidera
 beneficiare.
 
 ## Permessi, permessi straordinari e lavoro straordinario
 
-I permessi si dividono in ROL e EX-FESTIVITA' e possono essere richiesti anche nel corso della
-giornata lavorativa.
+Il numero di ore di permesso annuale varia in base all'anzianità di servizio.
+
+| Anzianità in mesi     | Ore/Anno |
+|-----------------------|----------|
+| 0 - 24                |    32    |
+| 25 - 48               |    60    |
+| >48                   |    88    |
+
+Un'altra tipologia di permessi retribuiti sono quelli che vengono concessi in casi straordinari.
+Ecco un elenco non esaustivo:
+
+- evento nascita
+- matrimonio
+- lutto parenti entro il secondo grado
+- donazione sangue
 
-Un'altra tipologia di permessi sono quelli che vengono concessi in casi straordinari come
-matrimonio, funerale, donazione del sangue etc…. Un elenco di permessi straordinari (retribuiti e
-non) è disponibile al
-seguente [link](http://www.fpcgil.it/flex/cm/pages/ServeBLOB.php/L/IT/IDPagina/7737).
+Se credi di avere diritto ad un permesso retribuito, scrivi al responsabile amministrativo il quale
+provvederà a fare un controllo per te.
 
-Ore di lavoro straordinario possono essere richieste in casi eccezionali dall'azienda per
-particolari esigenze di organizzazione del lavoro.
+Il tempo di lavoro straordinario è quello che eccede le 8 ore giornaliere. Per legge non si può,
+in nessun caso, superare la soglia delle 48 ore settimanali.
+
+### Compensazione e arrotondamento
+
+A fine mese verranno compensate tutte le ore di lavoro straordinario e di permesso del mese. Il
+saldo verrà poi arrotondato alla mezz'ora più vicina secondo le seguenti regole:
+
+- da 0 a 0,249 = 0
```

Comment:
Can you add ; after the each item? No need for the dot on the last one since you're continuing the sentence afterwards.